include/usb/usb_common.h

```diff
@@ -101,9 +101,11 @@
  * D4...0:Reserved -> 0
  */
 #define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP  0x20
```

Comment:
In fact USB_CONFIGURATION_ATTRIBUTES is bitmask so i would personally go for:

```diff
diff --git a/include/usb/usb_common.h b/include/usb/usb_common.h
index 632524a464..d2220a9692 100644
--- a/include/usb/usb_common.h
+++ b/include/usb/usb_common.h
@@ -100,9 +100,10 @@
  * D5:Remote Wakeup -> 0,
  * D4...0:Reserved -> 0
  */
-#define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP  0x20
-#define USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED   0xC0
-#define USB_CONFIGURATION_ATTRIBUTES ((COND_CODE_1(USB_SELF_POWERED, \
+#define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP	BIT(5)
+#define USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED	BIT(6)
+#define USB_CONFIGURATION_ATTRIBUTES BIT(7) \
+	| ((COND_CODE_1(USB_SELF_POWERED, \
 	   (USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED), (0)))   \
 	| (COND_CODE_1(CONFIG_USB_DEVICE_REMOTE_WAKEUP,         \
 	   (USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP), (0))))

```
As @jfischer-phytec-iot  mentioned BIT(7) must be set (historical reasons - could be found in usb20 spec)

Reply:
(+) s/USB_SELF_POWERED/CONFIG_USB_SELF_POWERED as @jfischer-phytec-iot mentioned. I forgot about that.

Reply:
this makes sense

Reply:
And on the parentheses around the zeo in the 3rd argument of COND_CODE_1 that seems pretty standard. I see it in other places in the repo. Sometimes the argument is even empty parentheses.